mmeds/database.py

```diff
@@ -1065,7 +1085,15 @@ def create_import_line(self, table, structure, columns, row_index):
                 try:
```

Comment:
Rather than handling the case where the [key_table] keys are null inside the ```try: ... except KeyError: ``` 
why not move the ```if ... else ``` outside the ```try: ..... except:``` 
I think it would look cleaner, since the ```if ... else ``` would be for ```line.append()``` rather than on whether or not to raise an error.

Reply:
Maybe cleaner. Less efficient though. For all but one table, this case won't come up. This way avoids slowing down all the other calls

Reply:
Makes sense, didn't realize it was just one table that was expected to get handled in the ```try ... catch```